test/unit/test-amp-img.js

```diff
@@ -665,4 +666,121 @@ describe('amp-img', () => {
       );
     });
   });
+
+  // Firefox misbehaves on Windows for this test because getBoundingClientRect
+  // returns 0x0 for width and height. Strangely Firefox on MacOS will return
+  // reasonable values for getBoundingClientRect if we add an explicit wait
+  // for laid out attributes via waitForElementLayout. If we change the test to
+  // test for client or offset values, Safari yields 0px measurements.
+  // For details, see: https://github.com/ampproject/amphtml/pull/24574
+  describe
+    .configure()
+    .skipFirefox()
+    .run('layout intrinsic', () => {
+      let browser;
+      beforeEach(() => {
+        fixture.iframe.height = 800;
+        fixture.iframe.width = 800;
+        browser = new BrowserController(fixture.win);
```

Comment:
The explicit waitForElementLayout call added here was to make FF work on Mac. Since we're skipping FF anyway, I'm okay with removing it to make the test simpler. Or we could just leave it in for it to work on FF MacOS. I'm not too strongly opinionated either way.